packages/scene-composer/src/components/panels/scene-components/motion-indicator/ColorEditor.tsx

```diff
@@ -14,7 +13,7 @@ import { ColorPicker } from '../../ColorPicker/ColorPicker';
 
 import { DataBindingEditor } from './DataBindingEditor';
 
-const ColorSwatch = styled.div<{ backgroundColor: string }>`
+const ColorSwatch = styled.div<{ backgroundColor?: string }>`
```

Comment:
Is this intended to allow a prop to override the values defined below?

Reply:
yeah, this component is used to show the `backgroundColor` passed in